src/opentime/rationalTime.cpp

```diff
@@ -205,55 +288,108 @@ RationalTime::from_timecode(
     return RationalTime{ double(value), rate };
 }
 
+static void
+set_error(std::string const& time_string,
+          ErrorStatus::Outcome code,
+          ErrorStatus* err)
+{
+    if (err) {
+        *err = ErrorStatus(
+            code,
+            string_printf(
+                "Input time string '%s' is an invalid time string",
```

Comment:
Should either this message be more generic, or this specific message passed in?  Or maybe the function name made more specific?  It seems very focused on invalid time string but has the name `set_error`

Reply:
ok, made set_error generic